readme.md

```diff
@@ -14,25 +14,12 @@ $ npm install electron-main-fetch
 
 ## Usage
```

Comment:
The docs should mention how it works. This is especially important as the consumer might be surprised to find extra hidden windows.